docs/Tutorials/OpenAPI.md

```diff
@@ -488,18 +673,226 @@ As JSON, this could look as follows:
 }
 ```
 
-## Swagger and ReDoc
+### oneOf, anyOf and allOf Keywords
+
+OpenAPI 3.x provides several keywords which you can use to combine schemas. You can use these keywords to create a complex schema or validate a value against multiple criteria.
+- oneOf – validates the value against exactly one of the subschemas
+- allOf – validates the value against all the subschemas
+- anyOf – validates the value against any (one or more) of the subschemas
```

Comment:
use `*` for bullet points rather than `-`, and ensure there's a blank new-line between the first bullet-point and the paragraph.

Replace `subschemas` with `sub-schemas`